strawberryfields/device.py

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-This module contains a class that represents the specifications of
-a device available via the API.
+This module contains a class that represents a device available via the API.
```

Comment:
Can we say "Xanadu Cloud" here to be more specific?
```suggestion
This module contains a class that represents a device on the Xanadu Cloud.
```